config/config.go

```diff
@@ -83,9 +83,23 @@ const (
 	// SettingEventExpirationTimeout sets the expiration timeout for stored
 	// events. After this time events will be removed from the storage.
 	SettingEventExpirationTimeout = "event_exp_timeout"
-	// SettingEventExpirationTimeoutDefault definse the defautl expiration
+	// SettingEventExpirationTimeoutDefault define the default expiration
 	// timeout for events.
-	SettingEventExpirationTimeoutDefault = "604800" //one week
+	SettingEventExpirationTimeoutDefault = "604800" // one week
+
+	// SettingWebhooksAsynchronous is the config key for enabling asynchronous
+	// processing of the webhook requests.
+	SettingWebhooksAsynchronous = "webhooks_asynchronous"
+	// SettingWebhooksAsynchronousDefault define the default value for the
+	// asynchronous processing of webook requests.
+	SettingWebhooksAsynchronousDefault = false
```

Comment:
Because it is a break in the API: before, processing was synchronous; now, it becomes asynchronous.
Based on your comment I have the impression you want it to be on by default, so I'm changing it that way.